ramls/edge-rtac.raml

```diff
@@ -9,7 +9,30 @@ documentation:
 
 schemas:
   holdings: !include holdings.xsd
+  batchHoldings: !include batch-holdings.xsd
+  errors: !include raml-util/schemas/error.schema
 
+/rtac:
+  displayName: RTAC
+  get:
+    description: Batch RTAC for the specified ids
+    responses:
+      200:
+        description: "Success"
+        body:
+          application/xml:
+            type: batchHoldings
+    queryParameters:
+      mms_id:
```

Comment:
A couple things on the naming of this parameter...

1)  If we're moving away from the legacy API endpoint path, why not ditch this oddity?  Would `ids` be a better fit here?
2)  It should probably be plural since it's expected that it will contain a comma-separated list of ids.

Reply:
We're adding a new one while supporting the legacy API

Reply:
my comment was about the naming of the parameter... mms_id is a carry-over from the legacy API.  Since "mms_id" doesn't really mean anything within the context of FOLIO (i.e. we don't refer to instance Ids as mms_ids anywhere else) I'm suggesting we just name this something more appropriate.

Reply:
e.g. `GET /rtac/batch?instanceIds=...`

Reply:
done